qemu/tests/cfg/guest_iommu_test.cfg

```diff
@@ -57,3 +57,6 @@
             smp = 10
         - intremap_off:
             iommu_intremap = off
+        - ats_off:
+            del virtio_dev_ats
+            virtio_dev_ats = off
```

Comment:
I don't think we need line 61 here, please double check it. Thanks.

Reply:
In iommu test loop, virtio_dev_ats = on is a global parameter and other cases don't need this scenario with virtio_dev_ats = off. So I need to del  virtio_dev_ats first to make sure virtio_dev_ats = off works.

Reply:
I double confirmed it, no need line 61 here. 
The results were passed.
 (1/3) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.guest_iommu_test.ats_off.q35: STARTED
 (1/3) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.guest_iommu_test.ats_off.q35: PASS (125.72 s)
 (2/3) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.guest_iommu_test.ats_off.scenario1.q35: STARTED
 (2/3) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.guest_iommu_test.ats_off.scenario1.q35: PASS (174.77 s)
 (3/3) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.guest_iommu_test.ats_off.scenario2.q35: STARTED
 (3/3) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.guest_iommu_test.ats_off.scenario2.q35: PASS (156.00 s)

Reply:
Tested without line 61 and virtio_dev_ats = off works.Deleted the line 61. Thanks.